### DISCUSSION TITLE
Detect tab selection

### DISCUSSION THREAD

**Discussion by beaker1977** [vmware-clarity/ng-clarity#706] [Q&A] [answered]
How can I detect if a clr-tab has been selected? I have content I need to display on one of my tabs that will be pulled from a database but the query will take 10+ plus seconds to display so I'd like to display a spinner in the interim however I don't know how to detect the tab has been selected so I can start the spinner.

**Comment by kevinbuhmann** [comment, marked as answer]
Please see the "tabs with two-way binding" section in the tabs documentation: https://clarity.design/documentation/tabs

There is a `clrIfActiveChange` event you can use: https://stackblitz.com/edit/clarity-light-theme-clr15-u7gwhp?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.ts

**Reply by beaker1977** [reply]
Perfect. Just what I was looking for. FYI though the tabs documentation page has no info on 'clrIfActiveChange '. 

Bob

**Reply by kevinbuhmann** [reply]
Yes, it does. The two-way binding `[(clrIfActive)]` implies a `[clrIfActive]` input and a `(clrIfActiveChange)` output.
